theme/js/components/dataset/resource/resource.vue

```diff
@@ -0,0 +1,190 @@
+<template>
+  <article
```

Comment:
During the PR, I fixed a HTML error due to articles used inside a <ul> but it broke the current list design